src/xcode/ENA/ENA/Source/HTTPClientCore/Service/Services/_Service+Default.swift

```diff
@@ -225,7 +238,7 @@ extension Service {
 		switch urlRequest(resource.locator, resource.sendResource, resource.receiveResource) {
 		case let .failure(resourceError):
 			Log.error("Creating url request failed.", log: .client)
-			retryOrDefaultValueOrFailureHandling(resource, .invalidRequestError(resourceError), nil, completion)
+			retryOrDefaultValueOrFailureHandling(resource, 0, .invalidRequestError(resourceError), nil, completion)
```

Comment:
0 is already default, not need to pass it here. (also for all other places)

Reply:
this only works with properties without the _ - I changed it